float64[3] xyz_delta_ref
---
bool success
string message
string error
